### PR TITLE
Knt1 258 update demo email classificator to use object store

### PR DESCRIPTION
**Pull request by JYisus**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: classificator/src/common/exitpoint/go.mod [hunk 1]
```diff
@@ -33,9 +35,11 @@ require (
...
+	golang.org/x/sync v0.1.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
 	golang.org/x/text v0.7.0 // indirect
-	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	gopkg.in/yaml.v2 v2.2.8 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/konstellation-io/kre-runners/kre-go/v4 => ./../../../../../runners/kre-go
```
Comment (alexrodfe) [inline classificator/src/common/exitpoint/go.mod:45]: watchout
```suggestion
```

File: classificator/src/common/repairs_handler/go.mod [hunk 2]
```diff
@@ -39,3 +39,5 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/yaml.v2 v2.2.8 // indirect
 )
+
+replace github.com/konstellation-io/kre-runners/kre-go/v4 => ./../../../../../runners/kre-go
```
Comment (alexrodfe) [inline classificator/src/common/repairs_handler/go.mod:43]: same here
```suggestion
```

File: classificator/src/common/stats_storer/go.mod [hunk 3]
```diff
@@ -33,9 +35,11 @@ require (
...
+	golang.org/x/sync v0.1.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
 	golang.org/x/text v0.7.0 // indirect
-	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	gopkg.in/yaml.v2 v2.2.8 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/konstellation-io/kre-runners/kre-go/v4 => ./../../../../../runners/kre-go
```
Comment (alexrodfe) [inline classificator/src/common/stats_storer/go.mod:45]:
```suggestion
```

File: classificator/src/go-classificator/email_classificator/go.mod [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd // indirect
+	golang.org/x/net v0.7.0 // indirect
+	golang.org/x/sync v0.1.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
+	golang.org/x/text v0.7.0 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+)
+
+replace github.com/konstellation-io/kre-runners/kre-go/v4 => ./../../../../../runners/kre-go
```
Comment (alexrodfe) [inline classificator/src/go-classificator/email_classificator/go.mod:46]:
```suggestion
```

File: classificator/src/go-classificator/etl/go.mod [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd // indirect
+	golang.org/x/net v0.7.0 // indirect
+	golang.org/x/sync v0.1.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
+	golang.org/x/text v0.7.0 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+)
+
+replace github.com/konstellation-io/kre-runners/kre-go/v4 => ./../../../../../runners/kre-go
```
Comment (alexrodfe) [inline classificator/src/go-classificator/etl/go.mod:46]:
```suggestion
```

File: classificator/src/go-classificator/etl/main.go [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+	batchSize := int(req.BatchSize)
+
+	err = ctx.SendEarlyReply(&proto.Response{
+		Message: fmt.Sprintf("Processing %d emails", len(emails)),
+	})
+	if err != nil {
+		return fmt.Errorf("error sending early reply: %w", err)
+	}
+
+	amountOfBatches := int(math.Ceil(float64(len(emails)) / float64(batchSize)))
```
Comment (alexrodfe) [inline classificator/src/go-classificator/etl/main.go:56]: Is this int result rounded up?

**Review by dfinteligenz — APPROVED** [review]

**Review by alexrodfe — APPROVED** [review]

**Pull request merged**